apisix/plugins/authz-keycloak.lua

```diff
@@ -42,7 +43,6 @@ local schema = {
             enum = {"urn:ietf:params:oauth:grant-type:uma-ticket"},
             minLength = 1, maxLength = 100
         },
-        audience = {type = "string", minLength = 1, maxLength = 100},
```

Comment:
This is now called `client_id` which is more consistent with the `opendid-connect` plugin as well.

Reply:
Maybe we can keep the old field and add a deprecated check?
https://github.com/apache/apisix/blob/bbbdf58d555ee89144b6923e751deabf7e0bbf15/apisix/init.lua#L463-L470

Reply:
@spacewander Possible, it's just a name. So we keep `client_id` as well and prefer it, if given?

Reply:
Yes. Maybe we can add a description on them:
https://github.com/apache/apisix/blob/bbbdf58d555ee89144b6923e751deabf7e0bbf15/apisix/schema_def.lua#L75

Reply:
Ok, will change accordingly. Will also add/fix more test cases soon.

Reply:
Ok, done.